spec/controllers/articles_controller_spec.rb

```diff
@@ -1,10 +1,10 @@
 RSpec.describe ArticlesController, type: :controller do
 
+  user = let(:user) { create(:user) }
```

Comment:
This is redundant as the `let` already declares the variable.
```suggestion
  let(:user) { create(:user) }
```

Reply:
Nice catch